### PR TITLE
chore: Update readme and add license

### PR DESCRIPTION
**Pull request by marc2332**

### CODE REVIEW

**Comment by marc2332** [comment]
to to: change to mozilla public license as suggested by *loading*

File: README.md [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
-Andromeda
-</h1>
-
-The simplest JavaScript engine for running JavaScript code and TypeScript code
-with no transpilation BS.
+# Andromeda 🌌
+<a href="https://freyaui.dev/"><img align="right" src="./assets/andromeda.svg" alt="Freya logo" width="150"/></a>
+
+
+[![Discord Server](https://img.shields.io/discord/1015005816094478347.svg?logo=discord&style=flat-square)](https://discord.gg/sYejxCdewG)
```
Comment (load1n9) [inline README.md:5]: ~~wrong discord~~

File: README.md [hunk 2]
```diff
@@ -1,8 +1,17 @@
...
+The simplest JavaScript and TypeScript runtime, fully written in [Rust 🦀](https://www.rust-lang.org/) and powered [Nova](https://trynova.dev/).
+
+> Note: ⚠️ This project is still in early stages and is not suitable for serious use.
+
+#### Running locally
+
+```sh
+cargo run -- run ./examples.main
+```
+
```
Comment (load1n9) [inline README.md:16]: should we add an install manual here?

File: README.md (hunk 1 above)
Comment (marc2332) [reply README.md]: :trollface:

File: README.md (hunk 2 above)
Comment (marc2332) [reply README.md:16]: There aren't any relases yet so I'd say we wait until we have that to add install docs

**Pull request merged**